kgbrules/status_rules.py

```diff
@@ -0,0 +1,99 @@
+"""Rules applied to the first line of the commit"""
+
+import re
+
+from .config import MAX_STATUS_LENGTH
+from .config import TYPES
+
+STATUS_PATTERN = re.compile(
+    r"^([^()\s]*)(\s*)\(([^()\s]*)\)(\s*):(\s*)(.*\w)(\s*)$")
+
+_TYPE_STRING = ", ".join(sorted(TYPES))
+
+
+def _is_merge(status_line):
+    return status_line.startswith("Merge pull request #")
+
+def _matches_pattern(status_line)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Parse error at line 17 column 33:

   11: join(sorted(TYPES))
   12: 
   13: 
   14: def _is_merge(status_line):
   15:     return status_line.startswith("Merge pull request #")
   16: 
  -->  def _matches_pattern(status_line)
   18:     match = STATUS_PATTERN.match(status_line)
   19:     if match is None:
   20:         return ("Conform status to `<type>(<scope>): <subject>` pattern\n" +
   21:                 "type and scope cannot contain spaces or parenthesis")
   22:     errors = [] [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.sagacify.com/coding_rules#rule_key=python%3AParsingError)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) invalid syntax [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.sagacify.com/coding_rules#rule_key=Pylint%3AE0001)